doc/nrf/releases/release-notes-latest.rst

```diff
@@ -113,16 +171,43 @@ The following list summarizes the most important changes inherited from upstream
   * Updated the Controller to only use control procedures supported by the peer.
   * Added support for the nRF52820 SoC.
   * Removed the legacy Controller.
+  * Implemented a function to remove auxiliary advertising sets (``ll_adv_aux_set_remove()``).
+  * Implemented a function to remove all primary channels and auxiliary channels of an advertising set (``ll_adv_aux_set_clear()``).
+  * Fixed overflow that could happen when using uninitialized PDU.
+  * Removed redundant :option:`CONFIG_BT_LL_SW_SPLIT` conditional.
+  * Enforced support for Read RSSI command if the Connection State is supported.
+  * Added missing aux acquire on periodic advertising.
+  * Updated the implementation to schedule non-overlapping sync PDUs.
+  * Fixed the handling of HCI commands for extended advertising.
+  * Added a terminate event for extended advertising.
+  * Filled the missing Periodic Advertising interval in the Extended Advertising Report when auxiliary PDUs contain Sync Info fields.
+  * Filled the referenced event counter of the Periodic Advertising SYNC_IND PDU into the Sync Info structure in the Common Extended Advertising Header Format.
+  * Switched HCI threads to use the kernel stack.
 
 * Bluetooth Mesh:
 
   * Removed the ``net_idx`` parameter from the Health Client model APIs because it can be derived (by the stack) from the ``app_idx`` parameter.
+  * Documented :ref:`Mesh Shell commands <zephyr:bluetooth_mesh_shell>`.
+  * Allowed to configure the advertiser stack size (see :option:`CONFIG_BT_MESH_ADV_STACK_SIZE`).
+  * Resolved a corner case where the segmented sending would be rescheduled before the segments were done sending.
+  * Fixed TX seg buffer with NULL pointer reference if enabled mesh-friendly.
```

Comment:
I don't understand what we're trying to say here.

Reply:
`mesh-friendly` is a BLE mesh concept, I beat `TX seg` is a well.
@trond-snekvik How to rewrite this?

Reply:
```suggestion
  * Fixed dangling transport segmentation buffer pointer when Friend feature is enabled
```